WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -3,21 +3,35 @@ import Foundation
 class ReaderCardsStreamViewController: ReaderStreamViewController {
     private var currentPage = 1
 
+    private let readerCardTopicsIdentifier = "ReaderTopicsCell"
+
+    private var cards: [ReaderCard]? {
+        content.content as? [ReaderCard]
+    }
+
     lazy var cardsService: ReaderCardService = {
         return ReaderCardService()
     }()
 
     override func viewDidLoad() {
         super.viewDidLoad()
         ReaderWelcomeBanner.displayIfNeeded(in: tableView)
+        tableView.register(ReaderTopicsCardCell.self, forCellReuseIdentifier: readerCardTopicsIdentifier)
     }
 
     // MARK: - TableView Related
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        if let posts = content.content as? [ReaderCard], let cardPost = posts[indexPath.row].post {
-            return cell(for: cardPost, at: indexPath)
-        } else {
+        guard let card = cards?[indexPath.row] else {
+            return UITableViewCell()
+        }
+
+        switch card.type {
+        case .post:
+            return cell(for: card.post!, at: indexPath)
```

Comment:
@yaelirub I know you hate force unwraps! 😝 But in this case, I already know that `post` isn't `nil`, so its safe to do that.

Reply:
lol fair enough